oxide-auth-async/src/endpoint/mod.rs

```diff
@@ -93,14 +93,14 @@ pub trait OwnerSolicitor<Request: WebRequest> {
     ) -> OwnerConsent<Request::Response>;
 }
 
-#[async_trait(?Send)]
-impl<T, Request: WebRequest> OwnerSolicitor<Request> for T
-where
-    T: oxide_auth::endpoint::OwnerSolicitor<Request> + ?Sized,
-{
-    async fn check_consent(
-        &mut self, req: &mut Request, pre_grant: &PreGrant,
-    ) -> OwnerConsent<Request::Response> {
-        oxide_auth::endpoint::OwnerSolicitor::check_consent(self, req, pre_grant)
-    }
-}
+// #[async_trait]
```

Comment:
I removed these blanket impl for now because it causes a wall of errors that clutter the screen